UiPath.FreeRdpClient/UiPath.FreeRdpWrapper/FreeRdpWrapper.cpp

```diff
@@ -12,10 +12,44 @@ using namespace FreeRdpClient;
 
 namespace FreeRdpClient
 {
-	struct instance_data
+
+	char* ConvToUtf8(BSTR source)
+	{
+		std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>, wchar_t> convToUTF8;
```

Comment:
Initializing convToUTF8 takes time, it's better to use the same instance for all conversions